test/MeasureReportBuilder.test.ts

```diff
@@ -22,36 +22,22 @@ const patient2Id = '08fc9439-b7ff-4309-b409-4d143388594c';
 const simpleMeasure = getJSONFixture('measure/simple-measure.json') as fhir4.Measure;
 const ratioMeasure = getJSONFixture('measure/ratio-measure.json') as fhir4.Measure;
 const cvMeasure = getJSONFixture('measure/cv-measure.json') as fhir4.Measure;
+const cvMeasureScoringOnGroup = getJSONFixture('measure/group-score-cv-measure.json');
 
-const simpleMeasureBundle: fhir4.Bundle = {
-  resourceType: 'Bundle',
-  type: 'collection',
-  entry: [
-    {
-      resource: simpleMeasure
-    }
-  ]
-};
-
-const ratioMeasureBundle: fhir4.Bundle = {
-  resourceType: 'Bundle',
-  type: 'collection',
-  entry: [
-    {
-      resource: ratioMeasure
-    }
-  ]
-};
-
-const cvMeasureBundle: fhir4.Bundle = {
-  resourceType: 'Bundle',
-  type: 'collection',
-  entry: [
-    {
-      resource: cvMeasure
-    }
-  ]
-};
+function buildTestMeasureBundle(measure: fhir4.Measure): fhir4.Bundle {
+  return {
+    resourceType: 'Bundle',
+    type: 'collection',
+    entry: [
+      {
+        resource: measure
+      }
+    ]
+  };
+}
+const simpleMeasureBundle: fhir4.Bundle = buildTestMeasureBundle(simpleMeasure);
+const ratioMeasureBundle = buildTestMeasureBundle(ratioMeasure);
+const cvMeasureBundle: fhir4.Bundle = buildTestMeasureBundle(cvMeasure);
```

Comment:
```suggestion
const simpleMeasureBundle = buildTestMeasureBundle(simpleMeasure);
const ratioMeasureBundle = buildTestMeasureBundle(ratioMeasure);
const cvMeasureBundle = buildTestMeasureBundle(cvMeasure);
```

Should prob just use the inference here in all 3 cases